R/Design-methods.R

```diff
@@ -209,7 +209,7 @@ setMethod("simulate",
              truth, args = NULL, firstSeparate = FALSE,
              mcmcOptions = McmcOptions(),
              parallel = FALSE, nCores =
-               min(parallel::detectCores(), 5),
+               min(parallel::detectCores(), 5), derive = list(),
```

Comment:
`derive` needs to be added to the to the description above
`@param derive A list of functions that derive statistics from the target dose samples.`